examples/pong/pong.rs

```diff
@@ -18,9 +20,22 @@ impl<'a, 'b> State<GameData<'a, 'b>> for Pong {
         let StateData { world, .. } = data;
         use audio::initialise_audio;
 
+        // Load the spritesheet necessary to render the graphics.
+        let spritesheet = {
```

Comment:
world.exec possible?

Reply:
Why?

Reply:
1) Cleaner
2) Torkley will ask you to add it, like he did on my pr ;)

Reply:
The thing is it then requires to introduce it in the tutorial before systems, which I feel is inappropriate...